tests/serialize/data.js

```diff
@@ -99,7 +99,7 @@ var doNext = function (remaining) {
             return
           }
           doc = $rdf.sym($rdf.uri.join(right, base))
-          if (doc.uri.slice(0, 8) !== 'file:///') {
+          if (doc.uri.slice(0, 7) !== 'file://') {
```

Comment:
I propose we migrate this discussion to #168. URI validation is something which has come up several times and is separate from the work here migrating `Makefile` scripts to the `package.json`.